src/lib.rs

```diff
@@ -1198,6 +1198,10 @@ impl Api {
         Api { table: new_table }
     }
 
+    pub fn close(&mut self) {
```

Comment:
Why does `close` need to be explicitly exposed?

Reply:
I thought to add it since Connection & Stream also have explicit close, but anyway I've removed it.